src/tests/MainSnapshot.test.ts

```diff
@@ -6,11 +6,12 @@ import { expect } from "chai";
 import * as fs from "fs";
 import * as path from "path";
 
-import { Id64Array } from "@bentley/bentleyjs-core";
```

Comment:
Since we are removing MainSnapshot.ts (which I like btw) we should rename these tests DataExporter.test.ts to match what it is actually testing